recipes/r-metstat/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = '1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-metstat
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/MetStaT_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/MetStaT/MetStaT_{{ version }}.tar.gz
+  sha256: 77c25984193ebbaee80596ebfedc9bf9c1c142ba0a25d542a99376144d7f0010
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-mass
+    - r-abind
+    - r-pls
+  run:
+    - r-base
+    - r-mass
+    - r-abind
+    - r-pls
+
+test:
+  commands:
+    - $R -e "library('MetStaT')"           # [not win]
+    - "\"%R%\" -e \"library('MetStaT')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=MetStaT
+  license: Apache (== 2.0)
```

Comment:
The most canonical string here would be the short identifier according to spdx: `Apache-2.0` ... see https://spdx.org/licenses/Apache-2.0.html ... just to consider for the future ... we don't have this consistent in all the other r feedstocks...

Reply:
Thanks!